vms/platformvm/txs/fee/complexity_test.go

```diff
@@ -71,6 +101,30 @@ func BenchmarkTxComplexity(b *testing.B) {
 	}
 }
 
+func BenchmarkTxComplexity_Batch(b *testing.B) {
+	require := require.New(b)
+
+	unsignedTxs := make([]txs.UnsignedTx, 0, len(txTests))
+	for _, test := range txTests {
+		if test.expectedComplexityErr != nil {
+			continue
+		}
+
+		txBytes, err := hex.DecodeString(test.tx)
+		require.NoError(err)
+
+		tx, err := txs.Parse(txs.Codec, txBytes)
+		require.NoError(err)
+
+		unsignedTxs = append(unsignedTxs, tx.Unsigned)
+	}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		_, _ = TxComplexity(unsignedTxs...)
```

Comment:
do we need to assign the outputs to global variables so they are not optimized out?

Reply:
afaict the benchmark is running as expected